models/evc.py

```diff
@@ -420,33 +436,54 @@ def archive(self):
         """Archive this EVC on deletion."""
         self.archived = True
 
-    def _use_uni_vlan(self, uni: UNI):
+    def _use_uni_vlan(
+        self,
+        uni: UNI,
+        uni_dif: Union[None, UNI] = None
+    ):
         """Use tags from UNI"""
         if uni.user_tag is None:
             return
         tag = uni.user_tag.value
+        if not tag or isinstance(tag, str):
+            return
         tag_type = uni.user_tag.tag_type
-        if isinstance(tag, int):
-            result = uni.interface.use_tags(
-                self._controller, tag, tag_type
-            )
-            if not result:
-                intf = uni.interface.id
-                raise ValueError(f"Tag {tag} is not available in {intf}")
+        if (uni_dif and isinstance(tag, list) and
+                isinstance(uni_dif.user_tag.value, list)):
+            tag = range_difference(tag, uni_dif.user_tag.value)
+            if not tag:
+                return
+        uni.interface.use_tags(
+            self._controller, tag, tag_type, False
```

Comment:
Can you double this `False` arg here, I believe `use_lock=True` should still be in place here, I'm not sure if you also meant to use `check_order`

Reply:
Yes, you are correct, my mistake. Fixed.